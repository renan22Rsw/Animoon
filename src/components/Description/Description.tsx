import React from "react";
import SubTitles from "../Titles/SubTitles";

const Description = ({ description }: { description: string }) => {
  return (
    <div className="md:hidden  my-8 ">
      <SubTitles>Description</SubTitles>
      <div className="bg-[#1C1C1C] w-[90%] mx-auto p-8 ">
        <p className="text-white text-sm">{description}</p>
      </div>
    </div>
  );
};

export default Description;
