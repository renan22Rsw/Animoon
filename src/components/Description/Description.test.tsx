import { render, screen } from "@testing-library/react";
import Description from "./Description";

describe("Description component", () => {
  it("should render the description", () => {
    const descriptionText: string = "anime or manga description";
    render(<Description description={descriptionText} />);
    expect(screen.getByText(descriptionText)).toBeInTheDocument();
    expect(screen.getByText("Description")).toBeInTheDocument();
  });

  it("should match a snapshot", () => {
    const descriptionText: string = "anime or manga description";
    const { container } = render(<Description description={descriptionText} />);
    expect(container).toMatchSnapshot();
  });
});
