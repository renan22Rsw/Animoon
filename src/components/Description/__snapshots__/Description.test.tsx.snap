// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description component should match a snapshot 1`] = `
<div>
  <div
    class="md:hidden  my-8 "
  >
    <div
      class="m-4 px-4 text-[#EDEDED]"
    >
      Description
    </div>
    <div
      class="bg-[#1C1C1C] w-[90%] mx-auto p-8 "
    >
      <p
        class="text-white text-sm"
      >
        anime or manga description
      </p>
    </div>
  </div>
</div>
`;
